pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -798,3 +800,52 @@ func TestFlowExporter_findFlowType(t *testing.T) {
 		assert.Equal(t, tc.expectedFlowType, flowType)
 	}
 }
+
+func TestFlowExporter_fillEgressInfo(t *testing.T) {
```

Comment:
could we have some unit test coverage for the new code in `exportConn`?

Reply:
Sure, I took some time to look into it, and feel not very easy to add unit tests here. Currently, the unit tests for the `findFlowType` function don't cover the case of `ToExternal` flows. We will need to add an interface to the node route controller and mock it here for these test cases. Can I do this in a follow-up PR?

Reply:
sounds good